### PR TITLE
Annahprice patch 1

### PR DESCRIPTION
**Pull request by annahprice**
Hi both,

Could you please review this and let me know what you think? I'll merge after you've approved. After the interest we received in some of these guides/templates from various people at government I thought it would be good to have our toolkit in one public place - hopefully it will also be useful to share this page internally too. 

There are a few "coming soon" items - some of those (e.g. chart and dashboard guidance) are because they don't exist yet, but some (e.g. excel table tips) are because I'm not sure the best way to publish/link to the item. Once we have a first draft with these placeholders I'll speak to the rest of the team about adding in everything else we have.

Thanks! :smile: 
Anna

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
 # Toolkit
-Templates and resources for transforming publications
+
+The Transforming Publishing team have created a toolkit of templates, guides and resources to support the modernisation of both the back end production of statistical products ([Reproducible Analytical Pipelines](#reproducible-analytical-pipelines)) and the user-facing output of statistical products to a digital platform ([Transformed Publications](#transformed-publications)). These tools will also be helpful when moving from a proprietary software such as SPSS or SAS, to an open source software such as R for any statistical analyses. 
+
+## Reproducible Analytical Pipelines
+Reproducible Analytical Pipelines are a way to introduce a robust, efficient and reproducible work flow when carrying out data analysis. They have traditionally been applied to statistical publications, but the
```
Comment (esthem01) [inline README.md]: What happened here it trails off?

Comment (esthem01) [inline README.md]: ...principles can be applied for any analysis.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,32 @@
...
+- Use this [**blank R project template**](https://github.com/Health-SocialCare-Scotland/r-project-structure) when starting new R code to ensure you have a sensible workflow and structure. *Coming soon*: a package which allows you to create projects with this structure from within RStudio.
+- Use these [**National Statistic publication templates**](https://github.com/NHS-NSS-transforming-publications/RMarkdown) to automate your PDF reports using Rmarkdown. 
+- *Coming soon*: Tips on automating excel tables
+
+### Further resources
+- [**This app**](https://scotland.shinyapps.io/nhs-r-resources/) contains a range of resources for R, such as links to online training and cheatsheets, and information about internal R user groups.
+- The Transforming Publishing team also have a [**resources area**](https://github.com/NHS-NSS-transforming-publications/resources), with useful resources on technical issues such as version control.
+- If using git for version control, you may like to follow our team's [**GitHub guidance**](https://github.com/NHS-NSS-transforming-publications/GitHub-guidance) on workflow and security.
```
Comment (esthem01) [inline README.md]: Would change ...may like to .....to SHOULD like to.....  (ideally I would love to say MUST but that might be a step too far!!)

**Review by esthem01 — COMMENTED** [review]
Looks great Anna - well done

**Comment by annahprice** [comment]
@davidc92 I'm going to merge this so I can start sharing the toolkit, but please do take a look at the content as I'd appreciate any comments you have when you're back in the office.

**Pull request merged**